apps/site/assets/css/_schedule-page-line-diagram.scss

```diff
@@ -141,6 +141,14 @@ $line-width: 8px; // width of the colored line in the diagram
   }
 }
 
+.m-schedule-table-crowding {
+  margin-left: .5rem;
+}
+
+.m-schedule-diagram__prediction-crowding span {
+  vertical-align: text-top;
```

Comment:
You might want to try using the `icon-size-inline` mixin instead to get the desired alignment, like `@include icon-size-inline(1.125em, .21em);` but with better values